opm/autodiff/FullyImplicitBlackoilSolver.hpp

```diff
@@ -194,6 +194,12 @@ namespace Opm {
         std::vector<int>         primalVariable_;
 
         // Private methods.
+
+        // return true if wells are available
+        bool wellsActive() const { return wells_ ? wells_->number_of_wells > 0 : false ; }
+        // return wells object
+        const Wells& wells () const { assert( wells_ ); return *wells_; }
```

Comment:
I meant an example where assert( wells_ ) would not work. I haven't come across any in the last 12 years I haven been working with C++ (different version on different platforms).